src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, {useMemo} from 'react';
 import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
```

Comment:
I believe we're trying to stop using underscore in favor of Lodash. Can we do what we need with Lodash instead?

Reply:
Sure, let's use `lodash` 🙂

Reply:
Done - ce2907d.